BUILDING.md

```diff
@@ -0,0 +1,42 @@
+# Document outlining the new build process for the analyzere-python repository
+Dockerfile.base contains the base level dependencies for running the
+tests against the python client.
+
+Dockerfile is the image that inherits from the base image and runs the
+actual tests, so it doesn't have to rebuild the image each time.
+
+For creating a new baseline image, you will edit the Dockerfile.base,
+when you are happy with your changes, you can then retag it as the
+latest and push it to ECR. In this example we will just tag it latest,
+but you could push a versioned tag up to ECR if you like.
+
+*make your changes*
+```bash
+vi Dockerfile.base
+```
+
+*login to AWS*
+```bash
+$(aws ecr get-login --no-include-email --region us-east-1)
```

Comment:
Unknown options: --no-include-email

Reply:
Works for me, maybe I have a newer version of the awscli. This also assumes you have run `aws configure` and setup your access key and secret in there. I can add a documentation bit about that if you like.